### PR TITLE
FontAwesome 5 + CSS spinner

### PR DESCRIPTION
**Pull request by hypeJunction**

### CODE REVIEW

File: views/default/output/icon.php [hunk 1]
```diff
@@ -9,68 +9,11 @@
...
-	"thumbs-up-alt" => "thumbs-up",
-	"trash" => "trash-o",
-	"twitter" => "twitter-square",
-	"undo" => "reply",
-	"video" => "film"
-];
+$class = elgg_extract_class($vars, ['elgg-icon']);
 
-$convert = (bool) elgg_extract('convert', $vars, true);
+$covert = elgg_extract('convert', $vars, true);
```
Comment (jdalsem) [inline views/default/output/icon.php]: covert = convert

**Comment by jdalsem** [comment]
LGTM... are FA5 icons free to use/distribute?

**Comment by hypeJunction** [comment]
https://fontawesome.com/license

**Review by jeabakker — APPROVED** [review]

**Comment by jdalsem** [comment]
Do we need to mention this in docs somewhere? At least in the appendix i think? Or are generated releases notes enough?

**Comment by hypeJunction** [comment]
Shoot. I accidentally squashed the commits.

**Pull request merged**